trk/src/Components/TapHistory.js

```diff
@@ -3,11 +3,12 @@ import ListHistory from './ListHistory';
 import ClimbItem from './ClimbItem';
 
 const TapHistory = (props) => {
+    console.log('[TEST] TapHistory called');
     return (
         <ListHistory
             data={props.climbsHistory}
-            renderItem={(climb) => <ClimbItem climb={climb} tapId={climb.tapId} />}
-            keyExtractor={(climb, index) => index.toString()}
+            renderItem={(item) => <ClimbItem climb={item} tapId={item.tapId} tapTimestamp={item.tapTimestamp} fromHome={props.fromHome} />}
+            keyExtractor={(item, index) => index.toString()}
```

Comment:
prop drilling timestamp + renaming (climb) to (item). "Climb" was confusing as I was accessing the timestamp from the tap not the climb, yet it was called 'climb'